execution_workflows/APAtrap/samplesheet_example_files.csv

```diff
@@ -0,0 +1,3 @@
+sample,bam,bai,bed
+Control,path_to/siControl_R1_2genes.bam,path_to/siControl_R1_2genes.bam.bai,path_to/vm18.mouse.genemodel.bed
+Srsf3,path_to/siSrsf3_R1_2genes.bam,path_to/siSrsf3_R1_2genes.bam.bai,path_to/vm18.mouse.genemodel.bed
```

Comment:
ohh sorry I don't think I understand completely! When I developed this locally I thought I had provide absolute paths to these test files. I went to the folder with these files, run pwd, and then replace path_to with the path shown by pwd. If we have path_to/../../tests/test_data/, it looks like we're going to the right folder, and then we go out of it, and go back into test_data, unless I'm interpreting this incorrectly 😆 What is the ../../tests/test_data for?

Reply:
> I went to the folder with these files


Oooh sorry this is my bad, this is what I didn't do! I was sitting in the APAtrap directory, typed `pwd` and used that to replace `path_to/`, which I obviously realised wouldn't work as we don't have the test data in this directory. I thought it would be better to just use the relative paths so I tested that first, not knowing that Nextflow struggles with relative paths. My workaround works like this

```
# relative path to file is ../../tests/test_data/siControl_R1_2genes.bam
# in sample sheet I had path_to/../../tests/test_data/siControl_R1_2genes.bam
$ pwd
/home/APAeval/execution_workflows/APAtrap

# replace path_to with `/home/APAeval/execution_workflows/APAtrap`
$ ls /home/APAeval/execution_workflows/APAtrap/../../tests/test_data/siControl_R1_2genes.bam
 /home/APAeval/execution_workflows/APAtrap/../../tests/test_data/siControl_R1_2genes.bam
```

Both are equally valid in my opinion:

1. Keep as is, but specify in README to change to the `tests/test_data` repo and type `pwd` (we know where it is but others may not).
2. Go with my suggestion, and specify that you should be in `execution_workflows/APAtrap` when you type `pwd`

Slight preference to 2nd option as I'd generally assume people wanting to run this workflow would already be in the APAtrap directory and this saves jumping about, but probably my personal bias creeping in :P

@ninsch3000 @faricazjj thoughts?